src/codegen/properties.rs

```diff
@@ -83,28 +85,63 @@ fn declaration(env: &Env, prop: &Property) -> String {
         let param_type = if let Some(Bound {
             alias,
             ref type_str,
+            ref bound_type,
             ..
         }) = prop.bound
         {
-            let value_bound = if !prop.is_get {
-                if *prop.nullable {
-                    " + glib::value::SetValueOptional"
-                } else {
-                    " + glib::value::SetValue"
+            use library::Type::*;
+            use analysis::bounds::BoundType;
+
+            let type_ = env.library.type_(prop.typ);
+            let bound_type = match *type_ {
+                Fundamental(_) => Some(bound_type.clone()),
+                _ => match bound_type {
+                    BoundType::Into(_, Some(ref x)) => Some(*x.clone()),
```

Comment:
Despite this branch only string (`&str`) properties converted.
This strange.
Ex. for me `fn set_property_cell_background_rgba<'a, P: Into<Option<&'a gdk::RGBA>>>(&self, cell_background_rgba: P) {` works in fine, and don't needed extra '+SetValueOptional` etc.

Reply:
Yes but for now we only use `Into` on `Fundamental` types and I'd like to keep it this way until we fully switch all `IsA` implementations.

Reply:
Ok